test/teos/unit/test_api.py

```diff
@@ -189,8 +189,10 @@ def test_register_json_no_inner_dict(client):
 
 
 def test_add_appointment(internal_api, client, appointment, block_processor):
-    # Simulate the user registration (end time does not matter here)
-    internal_api.watcher.gatekeeper.registered_users[user_id] = UserInfo(available_slots=1, subscription_expiry=0)
+    # Simulate the user registration (end time does not matter here  as long as is in the future)
```

Comment:
extra space, and typo "as long as it is"

Reply:
Fixed.